src/components/Dashboard/ThisWeek/WeeklyActivity/WeeklyActivity.tsx

```diff
@@ -252,7 +266,12 @@ const WeeklyActivity = ({ accountListId }: Props): ReactElement => {
             </Table>
           </TableContainer>
         </CardContent>
-        <CardActions sx={{ justifyContent: 'space-between' }}>
+        <CardActions
+          sx={{
+            justifyContent:
+              questions.length > 0 ? 'space-between' : 'flex-start',
```

Comment:
Do you think this should be `flex-end` so that the button is on the right like the other dashboard cards?

```suggestion
              questions.length > 0 ? 'space-between' : 'flex-end',
```